dotCMS/src/integration-test/java/com/dotcms/datagen/TestDataUtils.java

```diff
@@ -948,25 +942,44 @@ public static Contentlet getFileAssetContent(Boolean persist, long languageId) {
 
             //Test file
             final String testImagePath = "com/dotmarketing/portlets/contentlet/business/test_files/test_image1.jpg";
-            final File originalTestImage = new File(
-                    ConfigTestHelper.getUrlToTestResource(testImagePath).toURI());
-            final File testImage = new File(Files.createTempDir(),
-                    "test_image1" + System.currentTimeMillis() + ".jpg");
-            FileUtil.copyFile(originalTestImage, testImage);
+            return createFileAsset(testImagePath, folder, languageId, persist);
+        } catch (Exception e) {
+            throw new DotRuntimeException(e);
+        }
+    }
 
-            ContentletDataGen fileAssetDataGen = new FileAssetDataGen(folder, testImage)
-                    .languageId(languageId);
+    public static Contentlet getFileAssetSVGContent(Boolean persist, long languageId) {
 
-            if (persist) {
-                return ContentletDataGen.publish(fileAssetDataGen.nextPersisted());
-            } else {
-                return fileAssetDataGen.next();
-            }
+        try {
+            final Folder folder = new FolderDataGen().nextPersisted();
+
+            //Test file
+            final String testImagePath = "com/dotmarketing/portlets/contentlet/business/test_files/test_image.svg";
+            return createFileAsset(testImagePath, folder, languageId, persist);
         } catch (Exception e) {
             throw new DotRuntimeException(e);
         }
     }
 
+    private static Contentlet createFileAsset(final String testImagePath, final Folder folder, final long languageId, final boolean persist) throws Exception{
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883765)